notebooks/official/automl/automl_tabular_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1083 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        stage_1_tuning_result_artifact_uri=stage_1_tuning_result_artifact_uri,
</pre><p>Rename the variable used in the notebook so that it's different from the parameter value?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>There's a comment here for Max, should remove it for the official notebook</p>

Reply:
Removed